src/state/queries/messages/conversation.ts

```diff
@@ -37,9 +40,11 @@ export function useMarkAsReadMutation() {
       convoId,
       messageId,
     }: {
-      convoId: string
+      convoId?: string
```

Comment:
If at all possible I'd like to avoid this. If we don't have this data, we probably shouldn't be rendering the components that call this mutation 🤔